### PR TITLE
[82453] [82685] Huggingface model conversion continues on FE after pipeline canceled

### PR DESCRIPTION
**Pull request by as-suvorov**
### All Submissions:
* [x] Create a meaningful name of the PR with reference to JIRA. Example: *[43714] [NIGHTLY] Take correct tag for snyk scans*
* [x] JIRA tickets:
  * 82453 - Huggingface model conversion continues on FE after pipeline canceled
  * 82685 - Internal server error when retrieving model details
* [x] Set assignee to the PR.
* [x] Add corresponding labels to the PR. 
* [x] Set appropriate milestone to the PR.
* [ ] Added tests for new functionality (unit / e2e)
* [x] Followed code style guidelines:
  * Code covered with typings
  * No debug printing in production code
  * No commented blocks of code in production code

### Available commands in pull request comments

1. `/bump:client` - bump to latest **master** branch in **client** submodule
1. `/bump:client:releases/2021/4` - bump to latest **releases/2021/4** branch in **client** submodule
1. `/bump:client:yatarkan:feature/yt/feature-name` - bump to latest **feature/yt/feature-name** branch from **yatarkan** remote fork in **client** submodule
1. `/update` - merge base (target) branch into target branch and push changes

> **NOTE**:
> For more details, please visit the [DEVELOPER.md](development/docs/DEVELOPER.md)

### CODE REVIEW

File: client/src/app/modules/model-manager/components/hugging-face-import-ribbon-content/huggingface-model-details/huggingface-model-details.component.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
       *ngIf="!(loading$ | async); else loading"
       [innerHTML]="markdownHTML$ | async"
     ></div>
+    <div class="readme-not-found wb-text-8" *ngIf="error$ | async as error">No readme found.</div>
```
Comment (yatarkan) [inline client/src/app/modules/model-manager/components/hugging-face-import-ribbon-content/huggingface-model-details/huggingface-model-details.component.html:17]: Do we use `error` obtained from `error$` observable anywhere?

File: wb/main/utils/safe_runner.py [hunk 2]
```diff
@@ -25,28 +25,36 @@
 from wb.main.jobs.feed.feed_socket_service import FeedSocketService
 
 
+def safe_run_configurable(emit_web_socket: bool):
```
Comment (yatarkan) [inline wb/main/utils/safe_runner.py:28]: Why not to use the same decorator name with default `emit_on_error` argument set tot true?

File: client/src/app/modules/model-manager/components/hugging-face-import-ribbon-content/huggingface-model-details/huggingface-model-details.component.html (hunk 1 above)
Comment (as-suvorov) [reply client/src/app/modules/model-manager/components/hugging-face-import-ribbon-content/huggingface-model-details/huggingface-model-details.component.html]: fixed

File: wb/main/utils/safe_runner.py (hunk 2 above)
Comment (as-suvorov) [reply wb/main/utils/safe_runner.py]: fixed

**Review by apaniukov — APPROVED** [review]

**Review by yatarkan — APPROVED** [review]

**Pull request merged**